third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -1087,7 +1088,8 @@ struct FpToFpOpConversion
             llvm::isa<Float8E5M2Type>(dstElementType) ||
         llvm::isa<Float32Type>(srcElementType) &&
             llvm::isa<Float8E5M2Type>(dstElementType) &&
-            isaFamily != AMD::ISAFamily::CDNA4) {
+            (roundingMode != RoundingMode::RTNE ||
```

Comment:
Be specific here to check `== RTZ`?

Reply:
The hardware conversion only supports RTNE. It's safer to check != RTNE in case we have other rounding modes in the future.